src/badge/Badge.style.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+import { color } from '../_utils/branding'
+
+export const StyledBadge = styled.span`
+  & {
```

Comment:
to test, but no need to add `&` to increase specificity

Reply:
Done